Source/Core/Core/HW/GCMemcard/GCMemcard.h

```diff
@@ -79,6 +80,29 @@ enum class GCMemcardRemoveFileRetVal
   DELETE_FAIL,
 };
 
+enum class GCMemcardValidityIssues
+{
+  FAILED_TO_OPEN,
+  IO_ERROR,
+  INVALID_CARD_SIZE,
+  INVALID_CHECKSUM,
+  FREE_BLOCK_MISMATCH,
+  DATA_IN_UNUSED_AREA,
+  COUNT
+};
+
+class GCMemcardErrorCode
+{
+private:
+  std::bitset<static_cast<size_t>(GCMemcardValidityIssues::COUNT)> errors;
```

Comment:
```suggestion
  std::bitset<static_cast<size_t>(GCMemcardValidityIssues::COUNT)> m_errors;
```

We use the `m_` prefix for private class variables.